corehq/apps/cloudcare/static/cloudcare/js/formplayer/utils/utils.js

```diff
@@ -52,7 +52,12 @@ hqDefine("cloudcare/js/formplayer/utils/utils", function () {
     Utils.currentUrlToObject = function () {
         var url = Backbone.history.getFragment();
         try {
-            return Utils.CloudcareUrl.fromJson(Utils.encodedUrlToObject(url));
+            const cloudcareUrl = Utils.CloudcareUrl.fromJson(Utils.encodedUrlToObject(url));
+            for (const queryKey in cloudcareUrl.queryData) {
+                // retrieve query inputs from Utils object
+                cloudcareUrl.queryData[queryKey].inputs = Utils.getCurrentQueryInputs(queryKey);
```

Comment:
do we also need to set queryInputs here?

Reply:
I think "no" since it's already being set here https://github.com/dimagi/commcare-hq/blob/5669124cef39ec39045407913fb5d93bbd59bb34/corehq/apps/cloudcare/static/cloudcare/js/formplayer/utils/utils.js#L299 whenever we intentionally `setQueryData` on the cloudcareUrl object

Reply:
I'm looking at how currentUrlToObject is used [here](https://github.com/dimagi/commcare-hq/blob/a7a0f81ca4e2f026f0f0a1c0d0cd86c656ced6a6/corehq/apps/cloudcare/static/cloudcare/js/formplayer/app.js#L438-L444) and it accesses `.queryData`. In this case, the queryData.input would be missing right?

Reply:
I believe `queryData.inputs` should only be missing if it was never set in the first place, and in those cases we wouldn't expect to find it anyway.

Reply:
Ah I think you're right, I was getting this confused with something else. This looks good